src/Calculator/AboutFlyout.xaml

```diff
@@ -72,5 +73,21 @@
                     Click="FeedbackButton_Click"/>
 
         </StackPanel>
+        <RichTextBlock x:Name="AboutFlyoutContribute"
+                       Grid.Row="2"
+                       Grid.ColumnSpan="2"
+                       Margin="12,10,12,6"
```

Comment:
Nit: I think if we bump this up to 18, it will be vertically offset to match space between the hyperlinks and the feedback button above.

Reply:
Have updated GitHub link margin. Below are two screenshots of about page.

About page in full screen mode.
<img src="https://user-images.githubusercontent.com/10975439/101382551-14c28e80-3886-11eb-8989-d6223d954dc4.png" alt="About Page" width="250">

About page with calculator restored down to minimum size.
<img src="https://user-images.githubusercontent.com/10975439/101382223-b39abb00-3885-11eb-87bd-c5e8c47d5b94.png" alt="About Page" width="250">